src/plugins/ssh/index.ts

```diff
@@ -146,6 +145,36 @@ const accessPropagationGuard = (
   };
 };
 
+/**
+ * Parses and prints a chunk of SSH output to stderr.
+ *
+ * If debug is enabled, all output is printed. Otherwise, only selected messages are printed.
+ *
+ * @param chunkString the chunk to print
+ * @param debug true if debug output is enabled
+ */
+const parseAndPrintSshOutputToStderr = (
+  chunkString: string,
+  debug?: boolean
+) => {
+  const lines = chunkString.split("\n");
+
+  // SSH only prints the "Authenticated to" message if the verbose option (-v) is enabled
+  for (const line of lines) {
+    if (debug) {
+      print2(line);
+    } else {
+      if (line.includes("Authenticated to")) {
```

Comment:
I suppose capturing this message is important to detect when port-forwarding is established. Can you add a code comment about that please?

Reply:
Done